gears_cli/__main__.py

```diff
@@ -73,10 +93,20 @@ def print_res(res, res_id):
 @click.option('--port', default=6379, type=int, help='Redis port to connect to')
 @click.option('--user', default=None, help='Redis acl user')
 @click.option('--password', default=None, help='Redis password')
+@click.option('--ssl', default=False, type=bool, help='Use ssl')
+@click.option('--ssl-password', default=None, help='Passphrase for ssl key')
```

Comment:
You don't need default=None on options that aren't set. They should default to None when required=True isn't set.

Reply:
Copied from the other, I can change..

Reply:
Fixed